scipy/sparse/matrix_io.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+import scipy.sparse
+
+
+def save(file, matrix, compressed=True):
+    """ Save a sparse matrix.
+    
+    Save the sparse matrix into a file in ``.npz`` format.
+    If compressed, the resulting file is compressed.
```

Comment:
We can simplify this docstring a bit:
```python
"""Save a sparse matrix to a file using ``.npz`` format.

Parameters
----------
```

The extra lines of description aren't needed, and the `compressed` parameter is described just below.